test/unit/contentChunk.js

```diff
@@ -14,48 +15,45 @@ describe('contentChunk: ', () => {
   });
 
   after(function() {
+    db.collection('fs.files').deleteMany({})
```

Comment:
@bradsawadye All of the calls in this after function are asynchronous and need to be handled as such. We should await the completion of all of them.

Reply:
okay will sort it out